net-misc/onedrivegui/onedrivegui-1.1.1-r2.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+M_PN=OneDriveGUI
+
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=(python3_{9..13})
+
+inherit desktop distutils-r1
```

Comment:
From `pkgcheck --scan`:
```
  UnusedInherits: version 1.1.1-r2: unused eclass: desktop
```
A little bit hard to read through everything though ;)